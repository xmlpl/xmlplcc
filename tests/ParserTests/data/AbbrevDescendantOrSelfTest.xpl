node[] main() {
  node[] x;

  x//child;
  x/descendant-or-self::node()/child;
}
