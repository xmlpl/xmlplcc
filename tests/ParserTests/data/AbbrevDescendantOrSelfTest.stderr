4:4: error: unexpected token: //
error: input contains errors
